flake.nix

```
@@ -1,165 +1,174 @@
+        };
+
+        ruby = {
+          path = ./ruby;
+          description = "Flake for building ruby gems";
+          welcomeText = ''
+            # Simple Ruby Gem Template
+            ## Intended usage
+            The intended usage of this flake is to facilitate building ruby gems
+
+            ## More info
+            - [Ruby language](https://www.ruby-lang.org/)
+            - [Ruby in the NixOS manual](https://nixos.org/manual/nixpkgs/stable/#sec-language-ruby)
+          '';
+        };
+
+        rust-web-server = {
+          path = ./rust-web-server;
+          description = "A Rust web server including a NixOS module";
+        };
+
+        compat = {
+          path = ./compat;
+          description = "A default.nix and shell.nix for backward compatibility with Nix installations that don't support flakes";
+        };
+
+        haskell-hello = {
+          path = ./haskell-hello;
+          description = "A Hello World in Haskell with one dependency";
+        };
+
+        hercules-ci = {
+          path = ./hercules-ci;
+          description = "An example for Hercules-CI, containing only the necessary attributes for adding to your project.";
+        };
+
+        full = {
+          path = ./full;
+          description = "A template that shows all standard flake outputs";
+          welcomeText = ''
+            You just created a template that will show you all standard flake outputs.
+
+            Read more about it here:
+
+              https://github.com/NixOS/templates/tree/master/full
+          '';
+        };
+
+        pandoc-xelatex = {
+          path = ./pandoc-xelatex;
+          description = "A report built with Pandoc, XeLaTex and a custom font";
+        };
+
+        latexmk = {
+          path = ./latexmk;
+          description = "A simple LaTeX template for writing documents with latexmk";
+        };
+
+        go-hello = {
+          path = ./go-hello;
+          description = "A simple Go package";
+        };
+
+        empty = {
+          path = ./empty;
+          description = "A flake with no outputs";
+        };
+
+        haskell-nix = {
+          path = ./haskell.nix;
+          description = "An haskell.nix template using hix";
+          welcomeText = ''
+            You just created an haskell.nix template using hix. Read more about it here:
+            https://input-output-hk.github.io/haskell.nix/tutorials/getting-started-flakes.html
+          '';
+        };
+
+        haskell-flake = {
+          path = ./haskell-flake;
+          description = "A haskell-flake template";
+          welcomeText = ''
+            You just created a haskell-flake template.
+            See the README or https://github.com/srid/haskell-flake.
+          '';
+        };
+
+        utils-generic = {
+          path = ./utils-generic;
+          description = "Simple, all-rounder template with utils enabled and devShell populated";
+        };
```

This template serves the same purpose as the new multi-system one - keeping them both around doesn't make sense.

```suggestion
        utils-generic = self.templates.multi-system;
```

(Of course, the utils-generic template file should also be deleted - but I can't do that in a github review context.)

---

I agree that the `utils-generic` template (which I wasn't aware of previously) doesn't need to be there, especially given that using `flake-utils` for per-system outputs is unnecessary.